build-gluster-org/jobs/shellcheck.yml

```diff
@@ -9,41 +9,35 @@
     - glusterfs
 
     properties:
+    - github:
+        url: https://github.com/gluster/glusterfs
     - discard-after-x:
         x: 30
     - one-build-per-node
 
     triggers:
-    - gerrit:
-        trigger-on:
-          - patchset-created-event:
-              exclude-drafts: false
-              exclude-trivial-rebase: false
-              exclude-no-code-change: false
-          - draft-published-event
-          - comment-added-contains-event:
-              comment-contains-value: "recheck smoke"
-        server-name: review.gluster.org_for-smoke-jobs
-        skip-vote:
-          failed: true
-          unstable: true
-          notbuilt: true
-        projects:
-          - project-compare-type: 'PLAIN'
-            project-pattern: 'glusterfs'
-            branches:
-              - branch-compare-type: 'ANT'
-                branch-pattern: '**'
+    - github-pull-request:
+        cancel-builds-on-update: true
+        allow-whitelist-orgs-as-admins: true
+        admin-list:
+          - deepshikhaaa
+          - mscherer
+        org-list:
+          - gluster
+        github-hooks: true
+        only-trigger-phrase: false
+        trigger-phrase: 'recheck smoke'
+        skip-build-phrase: 'no tests'
+        permit-all: true
+        status-context: "Testing: shellcheck"
+        started-status: "Running: shellcheck"
+        success-status: "OK - shellcheck"
+        failure-status: "FAIL - please fix for merge"
 
     parameters:
-    - string:
-        default: refs/heads/master
-        description: 'For review 12345 and patch set 6, this will be refs/changes/45/12345/6. Default: Tip of master'
-        name: GERRIT_REFSPEC
-    - string:
-        default: master
-        description: 'Name of the branch you want to build from. We usually build from master'
-        name: GERRIT_BRANCH
+     string:
```

Comment:
A '-' is missing here, hence the traceback in jenkins. 
parameters is a list, and without the -, it make it a empty dict.

(I found it by replicating the jjb job on my laptop with a venv and a generous use of the magic "print random var until I get a clue" and seen it failed on shellcheck.yml content, if people wonder).